plugin_repackaging.sh

```
@@ -12,183 +12,155 @@ PIP_MIRROR_URL="${PIP_MIRROR_URL:-$DEFAULT_PIP_MIRROR_URL}"
-		fi
-	fi
-	cd ${CURR_DIR}
-	chmod 755 ${CURR_DIR}/${CMD_NAME}
-	${CURR_DIR}/${CMD_NAME} plugin package ${CURR_DIR}/${PACKAGE_NAME} -o ${CURR_DIR}/${PACKAGE_NAME}-${PACKAGE_SUFFIX}.difypkg --max-size 5120
-	if [ $? -ne 0 ]; then
-    echo "Repackage failed."
-    exit 1
-  fi
-	echo "Repackage success."
+    local PKG_PATH=$1
+    local PKG_FULL=$(basename "$PKG_PATH")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)